### DISCUSSION TITLE
Trasparent title bar

### DISCUSSION THREAD

**Discussion by Auties00** [bdlukaa/fluent_ui#927] [Q&A]
Hello,
I'm using bitsdojo_window's custom frame:
```
#include <bitsdojo_window_windows/bitsdojo_window_plugin.h>
auto bdw = bitsdojo_window_configure(BDW_CUSTOM_FRAME);
```

I noticed that when the acrylic effect from flutter_acrylic is used
```
    await Window.setEffect(
        effect: WindowEffect.acrylic,
        color: Colors.transparent,
        dark: SystemTheme.isDarkMode
    );
```
and the window is in focus, the title bar doesn't get completely hidden:
![image](https://github.com/bitsdojo/bitsdojo_window/assets/28218457/47333dee-573c-47e2-9e94-32cfc206f53f)
(Window out of focus)
![Screenshot (21)](https://github.com/bitsdojo/bitsdojo_window/assets/28218457/1a559ed7-b89a-4343-a150-74c1850c313b)
(Window in focus)

Does anyone know how I can achieve the effect I'm after?

**Comment by bdlukaa** [comment]
I'd suggest you to ask in the [`flutter_acrylic`](https://github.com/alexmercerind/flutter_acrylic) repository. Also, please note that the `bitsdojo_window` is a very old package and has not received proper maintenance through time - and Flutter has changed and improved a lot its Windows support. Consider not using it
